build/Targets/Versions.props

```diff
@@ -49,6 +49,8 @@
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' == 'Windows_NT'">$(UserProfile)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' != 'Windows_NT'">$(HOME)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+
+    <Version Condition="$(Version) == ''">$(NuGetPerBuildPreReleaseVersion)</Version>
```

Comment:
Downgrade warnings otherwise since referencing NETCore.App includes Roslyn. The projects are seen as 1.0.0 as default and the NETCore.App version is >= 1.3.0, so 1.0.0 is always a downgrade.